src/config/config-initializer.js

```diff
@@ -0,0 +1,33 @@
+// LICENSE : MIT
+'use strict';
+const Promise = require('bluebird');
+const fs = require('fs');
+const path = require('path');
+const Config = require('../config/config');
+
+// Public Interface
+/**
+ * Initializer class for config of textlint.
+ */
+const init = {
+    /**
+     * Create .textlintrc file in the current working directory
+     * @returns {int} The exit code for the operation.
+     */
+    initializeConfig() {
+        let rcFile = "." + Config.CONFIG_FILE_NAME + "rc";
+        let output = JSON.stringify({"rules":{}}, null, '  ');
```

Comment:
[JSON.stringify(value[, replacer[, space]])](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify "JSON.stringify(value[, replacer[, space]])")'s `space` options is String or Number.
I prefer Number. `JSON.stringify({"rules":{}}, null, 2);`